blazor/common/data-binding/dapper-using-webapp.md

```diff
@@ -0,0 +1,570 @@
+---
+layout: post
+title: Bind data using Dapper and perform CRUD operations | Syncfusion
```

Comment:
Web App can be added in both title and description

Reply:
Web App added in both title and description.